src/views/mfa-verify/PushForm.js

```diff
@@ -92,6 +94,10 @@ define(['okta', 'util/CookieUtil'], function (Okta, CookieUtil) {
       if (this.model.isValid()) {
         this.listenToOnce(this.model, 'error', this.setSubmitState, true);
```

Comment:
in case of error this.setSubmitState is called and you call it again after 30s as well. SetSubmitState would be called twice but we really need it just once.